Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -658,7 +658,7 @@ requestInfoCardView push state =
         [ height MATCH_PARENT
         , width MATCH_PARENT
         ]
-        [ RequestInfoCard.view (push <<< RequestInfoCardAction)  ]
+        [ RequestInfoCard.view (push <<< RequestInfoCardAction) (requestInfoCardConfig "dummyString") ]
```

Comment:
@aryanguptaSG  Please add the Function of LazyCheck here
Don't use dummy strings